client/src/components/RelatedItems/Outfits/Outfits.jsx

```diff
@@ -0,0 +1,148 @@
+import { useState, useEffect } from 'react';
+import { Rate } from 'antd';
+import { PlusOutlined, CloseCircleOutlined } from '@ant-design/icons';
+import { Virtual, Navigation, Pagination } from 'swiper/modules';
+import { Swiper, SwiperSlide } from 'swiper/react';
+import {
+  getRelatedIds,
+  getDetailById,
+  getReviewById,
+  getProductInfoById,
+} from '@/api/product-api.js';
+import 'swiper/css';
+import 'swiper/css/pagination';
+import 'swiper/css/navigation';
+const Outfits = () => {
+  const [swiperRef, setSwiperRef] = useState(null);
+  const [slides, setSlides] = useState([]);
+
+  useEffect(() => {
+    if (localStorage.slide) setSlides(JSON.parse(localStorage.slide));
+  }, []);
+
+  const handleRemoveOutfit = (id) => {
+    setSlides((pre) => {
+      return pre.filter((item) => item.id !== id);
+    });
+    localStorage.removeItem('slide');
+  };
+
+  const handleAddOutfit = async () => {
+    const allRes = await Promise.all([
+      getDetailById(37311),
+      getReviewById(37311),
+    ]);
+    const finalRes = [];
+
+    const goodsDetail = allRes[0].results[0];
```

Comment:
This is not working. You need to access the `data` property on the returned axios response to access .results and other JSON data. 
`allRes[0].data.results[0]`

Similar issue at multiple points in this code